pkg/cmd/grafana/apiserver/server.go

```diff
@@ -83,6 +83,7 @@ func (o *APIServerOptions) ModifiedApplyTo(config *genericapiserver.RecommendedC
 	if err := o.RecommendedOptions.Traces.ApplyTo(config.Config.EgressSelector, &config.Config); err != nil {
 		return err
 	}
+	o.RecommendedOptions.SecureServing.BindPort = 8443
```

Comment:
I did this to be able to run the API server without root privileges, maybe we can add proper support for this in dev environments?

Reply:
@charandas -- IIRC, you can use command line args to change the port you want to run on